packages/destination-actions/src/destinations/salesforce/sf-properties.ts

```diff
@@ -24,6 +24,16 @@ export const traits: InputField = {
   type: 'object'
 }
 
+export const custom_fields: InputField = {
```

Comment:
```suggestion
export const customFields: InputField = {
```

prefer camel case for consistency